src/applications/gi/sass/gi.scss

```diff
@@ -123,6 +123,18 @@
 }
 
 .vet-tec-programs-table {
+  .name-col {
+    width: 70%;
```

Comment:
Might be able to use the fixed column utilities here: https://design.va.gov/layout/flexbox-grid#fixed-columns maybe with a 5:1:1 ratio for .name-col : hours-col : tuition-col?